webapp/components/_new/features/SearchResults/SearchResults.vue

```diff
@@ -0,0 +1,338 @@
+<template>
+  <div id="search-results" class="search-results">
+    <div>
+      <ds-text class="total-search-results">
+        {{ $t('search.for') }} "
+        <strong>{{ search }}</strong>
+        "
+        <br />
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-09-23T06:42:09Z" title="Wednesday, September 23rd 2020, 8:42:09 am +02:00">Sep 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,328 @@
+<template>
+  <div id="search-results" class="search-results">
+    <div>
+      <ds-section>
+        <ds-text class="total-search-results">
+          <strong>{{ searchCount }}</strong>
+          {{ $t('search.results', {}, searchCount) }}
+        </ds-text>
```


Put the search string here a well? 🤔

I had situations where the the search string in the search field was not the same as for the results found. Happened as I went back `←` an forth `→` in the browser.

The text above the tab `X results found` is a bit technical in design as well with the frame around it. Is this even needed if we have the counts of the separate found types in the tabs?
But may be for the search string like I stated above.